app/templates/scripts/_webpack.base.js

```diff
@@ -118,29 +123,37 @@ module.exports = function(options) {
 
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify(options.env || 'development'),
-        '__APP_SERVER__': JSON.stringify(
-          options.serverPort ?
-            'http://localhost:' + options.serverPort :
-            options.cdn
-        ),
+        'process.env.PORT': JSON.stringify(serverPort),
       }),
 
       new webpack.optimize.DedupePlugin(),
 
       // TODO: optimize this by build
-      (options.env !== 'development' && new ExtractTextPlugin('styles.css'))
+      (options.env !== 'development' && new ExtractTextPlugin('styles.css')),
+      (options.env !== 'development' && new webpack.optimize.UglifyJsPlugin({
+        compress: {
+          warnings: false,
+        },
+      }))
     ),
 
     devtool: 'sourcemap',
 
-    rework: {
-      use: [
-        reworkLoader.plugins.imports,
-        reworkLoader.plugins.urls,
-      ],
+    postcss: function (webpack) {
+        return [
+            postcssImport({
+                addDependencyTo: webpack
+            }),
+            precss({
```

Comment:
is that only being used for vars?

Why not use something like https://github.com/MadLittleMods/postcss-css-variables to get the real CSS variable syntax?